[Bug]  Planck Rev7 fails to compile in web configurator

### Describe the Bug

Web configurator fails with error:

```
./.build/obj_planck_rev7_layout_ortho_4x12_mine/src/keymap.c:1,
                 from quantum/keymap_introspection.c:5:
./.build/obj_planck_rev7_layout_ortho_4x12_mine/src/keymap.c:17:253: error: 'EXT_PLV' undeclared here (not in a function)
```

Removing "EXT_PLV'" from the keymap results in a successful compile.

I'm not sure if the appropriate fix is to have the Planck firmware and default keymap updated, or to have the web configurator updated to support the keycode that is throwing the error here.


### Additional Context?

Using a fresh pull of the qmk firmware, and using a fresh copy of QMK_MSYS, version 1.7.2, the firmware compiles with no issue.

--- a/public/keymaps/p/planck_rev7_default.json
+++ b/public/keymaps/p/planck_rev7_default.json
@@ -38,7 +38,7 @@
       "KC_1",    "KC_1",    "KC_1",    "KC_1",    "KC_1",    "KC_1",    "KC_1",    "KC_1",    "KC_1",    "KC_1",    "KC_1",    "KC_1",
       "XXXXXXX", "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC",
       "XXXXXXX", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",
-      "EXT_PLV", "XXXXXXX", "XXXXXXX", "KC_C",    "KC_V",    "XXXXXXX", "XXXXXXX", "KC_N",    "KC_M",    "XXXXXXX", "XXXXXXX", "XXXXXXX"
+      "TO(0)",   "XXXXXXX", "XXXXXXX", "KC_C",    "KC_V",    "XXXXXXX", "XXXXXXX", "KC_N",    "KC_M",    "XXXXXXX", "XXXXXXX", "XXXXXXX"
     ],
     [
       "_______", "QK_BOOT", "DB_TOGG", "RGB_TOG", "RGB_MOD", "RGB_HUI", "RGB_HUD", "RGB_SAI", "RGB_SAD", "RGB_VAI", "RGB_VAD", "KC_DEL",

